shared/wallets/wallet-list/index.js

```diff
@@ -97,4 +111,43 @@ const AddWallet = FloatingMenuParentHOC(_AddWallet)
 
 const rowHeight = isMobile ? 56 : 48
 
-export {Wallet, AddWallet}
+type Props = {
+  wallets: Array<{
+    isSelected: boolean,
+    name: string,
+    keybaseUser: string,
+    contents: string,
+  }>,
+  onAddNew: () => void,
+  onLinkExisting: () => void,
+}
+
+type State = {
+  selectedWalletName: string,
+}
+
+class WalletList extends React.Component<Props, State> {
+  state: State = {
+    selectedWalletName: '',
+  }
+
+  render = () => {
+    return (
+      <Box2 direction="vertical" style={{height: '100%', width: 240}}>
+        {this.props.wallets.map(w => (
+          <Wallet
+            key={w.name}
+            onSelect={() => this.setState({selectedWalletName: w.name})}
+            isSelected={w.name === this.state.selectedWalletName}
```

Comment:
This needs to be in the store. It'll drive the right side of the screen

Reply:
Fixed!